nixpkgs_review/nix/evalAttrs.nix

```diff
@@ -1,8 +1,14 @@
-{ allowAliases ? false, attr-json }:
+{ allowAliases ? false, attr-json, extra-nixpkgs-config }:
 
 with builtins;
 let
-  pkgs = import <nixpkgs> { config = { checkMeta = true; allowUnfree = true; inherit allowAliases; }; };
+  pkgs = import <nixpkgs> ({
+    config = {
+      checkMeta = true;
+      allowUnfree = true; inherit allowAliases;
```

Comment:
If `extra-nixpkgs-config` contains `config` as an attribute, we're going to loose `allowUnfree`. We either need a recursive merge or we stick to `--nixpkgs-arg[str]` instead

Reply:
Here's another bit, I think: https://github.com/Mic92/nixpkgs-review/blob/6ffaffadeaafaf054fa5e76180789e387734fa6a/nixpkgs_review/buildenv.py#L38-L43